newui/docs/_data/navigation.yml

```diff
@@ -0,0 +1,51 @@
+
+# main links
+main:
+  - title: "Documentation"
+    url: /docs/quick-start-guide.html
+  - title: "Community"
+    url: /community.html
+  - title: "Roadmap"
+    url: /roadmap.html
+  - title: "Activities"
+    url: /activity.html
```

Comment:
can we show the apachehudi twitter stream here instead?

Reply:
> can we show the apachehudi twitter stream here instead?

Absolutely